src/c++/vdo/tests/OldRecoveryJournal_t1.c

```diff
@@ -50,7 +51,8 @@ static const char *RECOVERED = "testdata/vdo.old.rj.recovered";
 static char *crashedPath;
 static char *recoveredPath;
 
-static bool generateFiles = false;
+static bool  generateFiles = false;
+static char *expectedData = NULL;
```

Comment:
Nitpick: There would be no reason to change the format of line 54 if you put the new expectedData pointer with the other static char* variables, say after line 52.

Reply:
I put it here because its declared and defined, versus just declared as the other char * variables are.